graphene_django/rest_framework/tests/test_field_converter.py

```diff
@@ -96,8 +96,7 @@ def test_should_regex_convert_string():
 
 
 def test_should_uuid_convert_string():
-    if hasattr(serializers, "UUIDField"):
-        assert_conversion(serializers.UUIDField, graphene.String)
+    assert_conversion(serializers.UUIDField, graphene.String)
 
```

Comment:
DRF added support for `UUIDField` in [3.0.4](https://www.django-rest-framework.org/community/release-notes/#304) from 2015-01-28.  We [require DRF>= 3.6.3](https://github.com/graphql-python/graphene-django/blob/28c71c58f73e969bc05ebfdde06f578dd96e15d5/setup.py#L13)